portal/frontend/src/main/kotlin/com/sourceplusplus/portal/page/TracesPage.kt

```diff
@@ -52,4 +115,428 @@ class TracesPage(
             }
         }
     }
+
+    private fun setupUI() {
+        if (js("hideOverviewTab") as Boolean) {
```

Comment:
Any variables you access like this should be brought into the Kotlin page.
The same way `js("traceOrderType")` turned into:
```kotlin
class TracesPage(
    private val portalUuid: String,
    private val traceOrderType: TraceOrderType //<< this replaced js("traceOrderType")
)
```